README.md

```diff
@@ -12,21 +12,32 @@ Scroll to the bottom to get [instructions on how to use postman collection for t
 
 Read below to learn about ambrosus node development.
 
-## Install, setup, tests and linter
-To install dependencies:
+## Running a local development environment using Docker and docker-compose
+
+TODO: Instructions on how to generate the registry address and place it in the
+repository under config
```

Comment:
This needs to be clarified so that it's a simple as possible to setup a new environment.

Reply:
:+1: I think the whole section should be updated:D